vuepress/docs/docs/README.md

```diff
@@ -146,4 +146,4 @@ val subscriptions = Subscriptions(???)
 val api = graphQL(RootResolver(queries, mutations, subscriptions))
 ```
 
-All the fields of the subscription root case class MUST return `ZStream` or `? => ZStream` objects. When a subscription request is received, an output stream of `ResponseValue` will be returned wrapped in an `ObjectValue`.
+All the fields of the subscription root case class MUST return `ZStream` or `? => ZStream` objects. When a subscription request is received, an output stream of `ResponseValue` (a `StreamValue`) will be returned wrapped inside an `ObjectValue `.
```

Comment:
Oops, I inserted a spurious space before the backtick. Can you remove it?

Reply:
done.